docs/topics/practices.rst

```diff
@@ -102,6 +102,17 @@ reactor after ``MySpider`` has finished running.
     d.addBoth(lambda _: reactor.stop())
     reactor.run() # the script will block here until the crawling is finished
 
+.. note::
+    .. versionchanged:: VERSION
+
+    The Twisted reactor is now installed when
+    :meth:`~scrapy.crawler.CrawlerProcess.crawl` is first called, not when a
+    :class:`scrapy.crawler.CrawlerProcess` object is created. Because of this,
+    :setting:`TWISTED_REACTOR` and :setting:`ASYNCIO_EVENT_LOOP` are now
+    honored in :attr:`~scrapy.Spider.custom_settings`. In older Scrapy versions
+    they are silently ignored when set there and you need to set these settings
+    in some other way.
```

Comment:
I would prefer to move this to the release notes, to have the rest of the documentation only discuss what is current.

Maybe copy this _as is_ at the beginning of `news.rst` for now? (It is very well written, and we may forget to describe it in this much detail otherwise; we can adjust the wording later when we work on the final draft of the release notes)

(the same applies to the similar note in `settings.rst`)